extensions/amp-story-shopping/0.1/amp-story-shopping-tag.js

```diff
@@ -29,9 +29,16 @@ const FONTS_TO_LOAD = [
 /**
  * @param {!ShoppingConfigDataDef} tagData
  * @param {function(!ShoppingConfigDataDef): undefined} onClick
+ * @param {!AmpElement} element
+ * @param {?../../../src/service/localization.LocalizationService} localizationService
  * @return {!Element}
  */
-const renderShoppingTagTemplate = (tagData, onClick) => (
+const renderShoppingTagTemplate = (
+  tagData,
+  onClick,
+  element,
+  localizationService
```

Comment:
We can don't have to pass these as arguments if we put this method on the class. 
That way the scope will contain `this.element` and `this.localizationService_`.
Please update this to follow that pattern. [example](https://github.com/jshamble/amphtml/blob/shoppingInternationalizationCurrency/extensions/amp-story-share-menu/0.1/amp-story-share-menu.js#L190-L218).